uaclient/entitlements/fips.py

```diff
@@ -441,30 +442,34 @@ def incompatible_services(self) -> Tuple[IncompatibleService, ...]:
     def static_affordances(self) -> Tuple[StaticAffordance, ...]:
         static_affordances = super().static_affordances
 
-        fips_update = FIPSUpdatesEntitlement(self.cfg)
+        fips_updates = FIPSUpdatesEntitlement(self.cfg)
         enabled_status = ApplicationStatus.ENABLED
-        is_fips_update_enabled = bool(
-            fips_update.application_status()[0] == enabled_status
+        is_fips_updates_enabled = bool(
+            fips_updates.application_status()[0] == enabled_status
         )
 
+        services_once_enabled_obj = services_once_enabled_file.read()
         services_once_enabled = (
-            self.cfg.read_cache("services-once-enabled") or {}
+            {}
+            if not services_once_enabled_obj
+            else services_once_enabled_obj.to_json()
         )
-        fips_updates_once_enabled = services_once_enabled.get(
-            fips_update.name, False
+
+        fips_updates_once_enabled = (
+            fips_updates.name.replace("-", "_") in services_once_enabled
```

Comment:
I think this can all be replaced with
```python
services_once_enabled_obj = services_once_enabled_file.read()
fips_updates_once_enabled = services_once_enabled_obj.fips_updates if services_once_enabled else False
```